### ISSUE TITLE
Configure default schema for all entities

### ISSUE THREAD

**Issue by dmitryuk** [doctrine/DoctrineBundle#1562]
Locally I have two environments: `dev` and `test`.
While `dev` I use SQL dump with a lot of data to easier navigate while development.
While `test` I run tests. Before starting test the script do `schema:drop`, `schema:create` and running Doctrine fixtures.
Finally `test` environment deletes all data, restored in `dev` env.
So I want to configure different database schema (PostgreSQL)  only for `test` env to prevent deleting `dev`'s data.
But I did't find how to set it in symfony config:
```yaml
doctrine:
    dbal:
        connections:
            default:
                driver: '%database_driver%'
                host: '%database_host%'
                dbname: '%database_name%'
                user: '%database_user%'
                password: '%database_password%'
                # I want to set schema here
                #schema: test
```

**Comment by akuzia** [comment]
Schemas in psql are just namespaces, managed by search_path setting (public by default). So there is no difference in `public.table` and `table` outside of a default schema. Doctrine manages entity table names by annotating:

`@ORM\Table(name="dev.table")`
or
`@ORM\Table(schema="dev")`

In your case you can extend `Doctrine/DBAL/Driver`, provide your env, modify PDO object in connect method and `exec` `set search_path to ...`  command to accomodate your change.

_btw, consider separating your envs in different databases. This setup at first glance look kinda janky_

**Comment by ostrolucky** [comment]
This is out of scope of bundle, because DBAL doesn't allow to specify any other option than `dbame` for this, sorry
